manifests/z/Zheguisoft/IpchatGuiPc/1.0.0/Zheguisoft.IpchatGuiPc.yaml

```diff
@@ -0,0 +1,14 @@
+PackageIdentifier:  Zheguisoft.IpchatGuiPc
+PackageVersion:     1.0.0
+PackageLocale:      zh-CN
+Publisher:          Zheguisoft
+PackageName:        ipchat gui pc
+License:            Zheguisoft

```

Comment:
A username isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate